src/test/kotlin/coverosR3z/server/SocketTests.kt

```diff
@@ -194,10 +195,10 @@ class SocketTests() {
     fun testShouldGetHtmlFileResponseFromServer_badRequest() {
         // send a request to the server for something that doesn't exist
         val badRequests = listOf(
-                "GET BLAHBLAHBLAH HTTP/1.1\n\n",
-                "GET\n\n",
-                "BLAHBLAHBLAH HTTP/1.1\n\n",
-                "HTTP/1.1\n\n",
+                "GET BLAHBLAHBLAH HTTP/1.1\n",
+                "GET\n",
+                "BLAHBLAHBLAH HTTP/1.1\n",
+                "HTTPBYRON/1.1\n",
```

Comment:
Is it necessary to bungle the protocol for a request to be bad?

Reply:
Actually I left that in accidentally.  In fact, I added that because I was in the midst of getting confused, because the tests were impacting each other and I wasn't sure where "HTTP/1.1" was coming from, so I added my name to confirm it was definitely coming from that line.